ninetofiver/models.py

```diff
@@ -984,6 +986,7 @@ class Performance(BaseModel):
             validators.MaxValueValidator(31),
         ]
     )
+    redmine_time_entry_id = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
The moment a performance is created, it should no longer be linked to the import source. If you want a reference back, add a link in a description field. I don't want any redmine-specific fields here.